README.md

```diff
@@ -2,7 +2,37 @@
 
 A new programming language to be written in Swahili to help teach programming concepts to swahili speaking students.
```

Comment:
Since it's already written shouldn't it be "A programming language written in a swahili like syntax"